src/main/resources/data/bloodmagic/patchouli_books/guide/ru_ru/templates/crafting_alchemy_table.json

```diff
@@ -0,0 +1,109 @@
+{
+  "processor": "wayoftime.bloodmagic.compat.patchouli.processors.AlchemyTableProcessor",
+  "components": [
+    {
+      "type": "header",
+      "text": "#heading",
+      "x": -1,
+      "y": -6
+    },
+    {
+      "type": "text",
+      "text": "#text",
+      "x": 0,
+      "y": 51
+    },
+    {
+      "type": "image",
+      "image": "bloodmagic:textures/gui/patchouli_book/crafting.png",
+      "x": 2,
+      "y": 4,
+      "u": 0,
+      "v": 154,
+      "texture_width": 128,
+      "texture_height": 256,
+      "width": 62,
+      "height": 43
+    },
+    {
+     "type": "image",
+     "image": "bloodmagic:textures/gui/patchouli_book/crafting.png",
+      "x": 65,
+      "y": 4,
+      "u": 44,
+      "v": 111,
+      "texture_width": 128,
+      "texture_height": 256,
+      "width": 49,
+      "height": 43
+    },
+	{
+      "type": "item",
+      "item": "bloodmagic:alchemytable",
+      "x": 94,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#input1",
+      "x": 6,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#input2",
+      "x": 25,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#input3",
+      "x": 44,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#input4",
+      "x": 6,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#input5",
+      "x": 25,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#input6",
+      "x": 44,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#orb",
+      "x": 69,
+      "y": 27
+    },
+    {
+      "type": "tooltip",
+      "tooltip": ["Cost: #syphon# LP", "Duration: #time# Ticks"],
+      "x": 69,
+      "y": 8,
+      "width": 16,
+      "height": 16
+    },
+    {
+      "type": "text",
+      "text": "LP",

```

Comment:
This is the word that appears above the arrow in Alchemy Table recipes (to indicate a tooltip).  Need to translate "LP" (Life Points).